pkg/virtctl/create/clone/clone.go

```diff
@@ -188,6 +200,10 @@ func (c *createClone) newClone() (*clonev1alpha1.VirtualMachineClone, error) {
 		Target:            target,
 		AnnotationFilters: c.annotationFilters,
 		LabelFilters:      c.labelFilters,
+		Template: clonev1alpha1.VirtualMachineCloneSpecTemplate{
```

Comment:
Add test cases for this in clone_test.go?

Reply:
ok, I will add it